snort/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
+
+- name: Install prerequisites
+  apt: name="{{ item }}" state=present
+  with_items: snort_prerequisites
+  tags: ['snort', 'snort:install']
+
+- name: Create download directory
+  file: path="{{ snort_download_dir }}" state=directory mode=0755
+  tags: ['snort', 'snort:install']
+
+- name: Download md5 checksums
+  get_url: >
+    url="{{ snort_base_url }}/md5s"
+    dest="{{ snort_download_dir }}/md5s"
+  tags: ['snort', 'snort:install']
+
+- name: Download Snort DAQ
+  get_url: >
+    url="{{ snort_base_url }}/{{ snort_daq_archive }}"
+    dest="{{ snort_download_dir }}/{{ snort_daq_archive }}"
+  tags: ['snort', 'snort:install']
+
+- name: Download Snort
+  get_url: >
+    url="{{ snort_base_url }}/{{ snort_archive }}"
+    dest="{{ snort_download_dir }}/{{ snort_archive }}"
+  tags: ['snort', 'snort:install']
+
+- name: Verify checksum of Snort DAQ archive
+  shell: >
+    md5sum -c md5s 2>/dev/null | grep "{{ snort_daq_archive }}: OK"
+    chdir="{{ snort_download_dir }}"
+  tags: ['snort', 'snort:install']
+
+- name: Verify checksum of Snort archive
+  shell: >
+    md5sum -c md5s 2>/dev/null | grep "{{ snort_archive }}: OK"
+    chdir="{{ snort_download_dir }}"
+  tags: ['snort', 'snort:install']
+
+- name: Unarchive Snort DAQ
+  unarchive: >
+    src="{{ snort_download_dir  }}/{{ snort_daq_archive }}"
+    dest="{{ snort_download_dir }}"
+    copy=no
+  tags: ['snort', 'snort:install']
+
+- name: Unarchive Snort
+  unarchive: >
+    src="{{ snort_download_dir  }}/{{ snort_archive }}"
+    dest="{{ snort_download_dir }}"
+    copy=no
+  tags: ['snort', 'snort:install']
+
+- name: Install Snort DAQ
+  shell: >
+    ./configure && make && make install && ldconfig
```

Comment:
The package in the repos hasn't been updated for 2 years, and there doesn't seem to be a PPA. Building this from source is messy, but better than not being able to upgrade.